metric/system/cgroup/reader_test.go

```diff
@@ -33,8 +33,51 @@ import (
 const (
 	path = "/docker/b29faf21b7eff959f64b4192c34d5d67a707fe8561e9eaa608cb27693fba4242"
 	id   = "b29faf21b7eff959f64b4192c34d5d67a707fe8561e9eaa608cb27693fba4242"
+
+	pathv2 = "/system.slice/docker-1c8fa019edd4b9d4b2856f4932c55929c5c118c808ed5faee9a135ca6e84b039.scope"
+	idv2   = "docker-1c8fa019edd4b9d4b2856f4932c55929c5c118c808ed5faee9a135ca6e84b039.scope"
 )
 
+// func TestForPid(t *testing.T) {
+// 	pid := 3757
+// 	reader, err := NewReader(resolve.NewTestResolver(""), true)
+// 	assert.NoError(t, err, "error in NewReader")
+
+// 	vers, err := reader.CgroupsVersion(1989)
+// 	if vers == CgroupsV1 {
+// 		t.Logf("Got V1 process.")
+// 		stats, err := reader.GetV1StatsForProcess(pid)
+// 		assert.NoError(t, err, "error in GetV1StatsForProcess")
+// 		t.Logf("Event: %#v", stats)
+// 	}
+
+// 	if vers == CgroupsV2 {
+// 		t.Logf("Got V2 process.")
+// 		stats, err := reader.GetV2StatsForProcess(pid)
+// 		assert.NoError(t, err, "error in GetV2StatsForProcess")
+// 		t.Logf("Event: %#v", stats)
+// 	}
+
+// }
+
+func TestV1EventDifferentPaths(t *testing.T) {
+	pid := 3757
+	reader, err := NewReader(resolve.NewTestResolver("testdata/ubuntu1804"), true)
+	assert.NoError(t, err, "error in NewReader")
+
+	stats, err := reader.GetV1StatsForProcess(pid)
+	assert.NoError(t, err, "error in GetV1StatsForProcess")
```

Comment:
It's better to use `require` here so the test is interrupted. If we can't read/get the stats there is no point in continuing the tests.
```suggestion
	require.NoError(t, err, "error in NewReader")

	stats, err := reader.GetV1StatsForProcess(pid)
	require.NoError(t, err, "error in GetV1StatsForProcess")
```